mk/spksrc.native-env.mk

```diff
@@ -5,7 +5,7 @@ INSTALL_PREFIX = /usr/local
 
 # Unsetting variables MUST always be first
 # as otherwise it fails silently
-ENV := -u LDSHARED -u PKG_CONFIG -u PKG_CONFIG_LIBDIR -u PKG_CONFIG_PATH $(ENV)
+ENV := -u LDSHARED -u MAKEFLAGS -u PKG_CONFIG -u PKG_CONFIG_LIBDIR -u PKG_CONFIG_PATH $(ENV)
```

Comment:
If I get this right, re-adding the `-u MAKEFLAGS` solved `rabbitmq`?

Reply:
Yes it it fixes native erlang